File: dart/集合.dart
void main(){
  var arr={"html","css","java"};
 /*  print(arr is Set);
  print(arr); */
 /*  arr.add("react");
  print(arr);
  var list=List.from(arr);
  print(list); */
  var list=[1,2,3,3];
  var s=new Set();
  s.addAll(list);
  print(s is Set);
  print(List.from(s));
}

File: dart/集合.js
/* var s=new Set([1,2,3,3])
var arr=Array.from(s)
console.log(arr) */
var arr=[1,2,3,4];
var test=Array.from(arr);
arr.push(5);
 //Array.from() 可以进行一个深拷贝        
console.log(arr)      //[ 1, 2, 3, 4, 5 ]